apigee_edge.module

```diff
@@ -336,6 +337,31 @@ function apigee_edge_entity_base_field_info_alter(&$fields, EntityTypeInterface
   }
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter() for install_configure_form().
+ *
+ * Allows the profile to alter the site configuration form.
+ */
+function apigee_edge_form_install_configure_form_alter(&$form, FormStateInterface $form_state) {
+  $form['#validate'][] = 'apigee_edge_checkemail';
+}
+
+function apigee_edge_checkemail(&$form, FormStateInterface $form_state) {
```

Comment:
```suggestion
/**
*  @internal
*/
function _apigee_edge_site_install_form_check_email(array &$form, FormStateInterface $form_state): void {
```

The benefit is marking this as internal with all power, nobody should use this function outside of the module.

But it still deserves a better doc block.

Reply:
Noted, will apply the change

Reply:
Updated